docs/user-guides/desktop/desktop-connect-sync.md

```diff
@@ -0,0 +1,175 @@
+You can also connect to the SSH server in your workspace using any SSH client, such as OpenSSH or PuTTY:
+
+   ```shell
+   ssh your-workspace.coder
+   ```
+
+Any services listening on ports in your workspace will be available on the same hostname. For example, you can access a web server on port `8080` by visiting `http://your-workspace.coder:8080` in your browser.
+
+> [!NOTE]
+> Currently, the Coder IDE extensions for VSCode and JetBrains create their own tunnel and do not utilize the Coder Connect tunnel to connect to workspaces.
+
+### Ping your workspace
+
+<div class="tabs">
+
+### macOS
+
+Use `ping6` in your terminal to verify the connection to your workspace:
+
+   ```shell
+   ping6 -c 5 your-workspace.coder
+   ```
+
+### Windows
+
+Use `ping` in a Command Prompt or PowerShell terminal to verify the connection to your workspace:
+
+   ```shell
+   ping -n 5 your-workspace.coder
+   ```
+
+</div>
+
+## Sync a local directory with your workspace
+
+Coder Desktop file sync provides bidirectional synchronization between a local directory and your workspace.
+You can work offline, add screenshots to documentation, or use local development tools while keeping your files in sync with your workspace.
+
+1. Create a new local directory.
+
+   If you select an existing clone of your repository, Desktop will recognize it as conflicting files.
+
+1. In the Coder Desktop app, select **File sync**.
+
+   ![Coder Desktop File Sync screen](../../images/user-guides/desktop/coder-desktop-file-sync.png)
+
+1. Select the **+** in the corner to select the local path, workspace, and remote path, then select **Add**:
+
+   ![Coder Desktop File Sync add paths](../../images/user-guides/desktop/coder-desktop-file-sync-add.png)
+
+1. File sync clones your workspace directory to your local directory, then watches for changes:
+
+   ![Coder Desktop File Sync watching](../../images/user-guides/desktop/coder-desktop-file-sync-watching.png)
+
+   For more information about the current status, hover your mouse over the status.
+
+File sync excludes version control system directories like `.git/` from synchronization, so keep your Git-cloned repository wherever you run Git commands.
+This means that if you use an IDE with a built-in terminal to edit files on your remote workspace, that should be the Git clone and your local directory should be for file syncs.
+
+> [!NOTE]
+> Coder Desktop uses `alpha` and `beta` to distinguish between the:
+>
+> - Local directory: `alpha`
+> - Remote directory: `beta`
+
+### File sync conflicts
+
+File sync shows a `Conflicts` status when it detects conflicting files.
+
+You can hover your mouse over the status for the list of conflicts:
+
+![Desktop file sync conflicts mouseover](../../images/user-guides/desktop/coder-desktop-file-sync-conflicts-mouseover.png)
+
+If you encounter a synchronization conflict, delete the conflicting file that contains changes you don't want to keep.
+
+## Accessing web apps in a secure browser context
```

Comment:
Can we move this to a troubleshooting section. This is not something we should highlight. 

Also if we can use a compacted view for troubleshooting topics to prevent them from occupying a significant space on page